.gitignore

```diff
@@ -25,4 +25,10 @@ out/
 local.properties
 
 # Xls lock files.
-.~lock*
\ No newline at end of file
+.~lock*
+
+# Generated native files
+/src/test/resources/export/userHome/native/jansi/null/osx/libjansi.jnilib
+/src/test/resources/export/userHome/native/30/osx-amd64/libnative-platform.dylib
+/src/test/resources/export/userHome/native/30/osx-amd64/libnative-platform.dylib.lock
+/src/test/resources/export_then_import/export/userHome/native/
```

Comment:
I'll remove this

Reply:
I'll leave it after all. These files are generated by Apache POI on macOS when running the unit tests.

Reply:
I didn't notice it at first, but similar folders are alo generated with Unix OS:
```
/src/test/resources/export/userHome/cache/6.3/fileHashes
/src/test/resources/export/userHome/cache/6.3/generated-gradle-jars
```

Thus, it would safer to declare parent folders:
```
/src/test/resources/export/userHome
/src/test/resources/export_then_import
```

Reply:
By the way, is the last one really necessary? (`export_then_import`)

Reply:
I declared only `export/userHome` and removed `export_then_import`